frame/contracts/src/schedule.rs

```diff
@@ -492,63 +398,10 @@ impl Default for Limits {
 }
 
 impl<T: Config> Default for InstructionWeights<T> {
+	/// We price both `i64.const` and `drop` as `instr_i64const / 2`. The reason
+	/// for that is that we cannot benchmark either of them on its own.
 	fn default() -> Self {
-		Self {
-			version: 4,
-			fallback: 0,
-			i64const: cost_instr!(instr_i64const, 1),
-			i64load: cost_instr!(instr_i64load, 2),
-			i64store: cost_instr!(instr_i64store, 2),
-			select: cost_instr!(instr_select, 4),
-			r#if: cost_instr!(instr_if, 3),
-			br: cost_instr!(instr_br, 2),
-			br_if: cost_instr!(instr_br_if, 3),
-			br_table: cost_instr!(instr_br_table, 3),
-			br_table_per_entry: cost_instr!(instr_br_table_per_entry, 0),
-			call: cost_instr!(instr_call, 2),
-			call_indirect: cost_instr!(instr_call_indirect, 3),
-			call_per_local: cost_instr!(instr_call_per_local, 0),
-			local_get: cost_instr!(instr_local_get, 1),
-			local_set: cost_instr!(instr_local_set, 1),
-			local_tee: cost_instr!(instr_local_tee, 2),
-			global_get: cost_instr!(instr_global_get, 1),
-			global_set: cost_instr!(instr_global_set, 1),
-			memory_current: cost_instr!(instr_memory_current, 1),
-			memory_grow: cost_instr!(instr_memory_grow, 1),
-			i64clz: cost_instr!(instr_i64clz, 2),
-			i64ctz: cost_instr!(instr_i64ctz, 2),
-			i64popcnt: cost_instr!(instr_i64popcnt, 2),
-			i64eqz: cost_instr!(instr_i64eqz, 2),
-			i64extendsi32: cost_instr!(instr_i64extendsi32, 2),
-			i64extendui32: cost_instr!(instr_i64extendui32, 2),
-			i32wrapi64: cost_instr!(instr_i32wrapi64, 2),
-			i64eq: cost_instr!(instr_i64eq, 3),
-			i64ne: cost_instr!(instr_i64ne, 3),
-			i64lts: cost_instr!(instr_i64lts, 3),
-			i64ltu: cost_instr!(instr_i64ltu, 3),
-			i64gts: cost_instr!(instr_i64gts, 3),
-			i64gtu: cost_instr!(instr_i64gtu, 3),
-			i64les: cost_instr!(instr_i64les, 3),
-			i64leu: cost_instr!(instr_i64leu, 3),
-			i64ges: cost_instr!(instr_i64ges, 3),
-			i64geu: cost_instr!(instr_i64geu, 3),
-			i64add: cost_instr!(instr_i64add, 3),
-			i64sub: cost_instr!(instr_i64sub, 3),
-			i64mul: cost_instr!(instr_i64mul, 3),
-			i64divs: cost_instr!(instr_i64divs, 3),
-			i64divu: cost_instr!(instr_i64divu, 3),
-			i64rems: cost_instr!(instr_i64rems, 3),
-			i64remu: cost_instr!(instr_i64remu, 3),
-			i64and: cost_instr!(instr_i64and, 3),
-			i64or: cost_instr!(instr_i64or, 3),
-			i64xor: cost_instr!(instr_i64xor, 3),
-			i64shl: cost_instr!(instr_i64shl, 3),
-			i64shrs: cost_instr!(instr_i64shrs, 3),
-			i64shru: cost_instr!(instr_i64shru, 3),
-			i64rotl: cost_instr!(instr_i64rotl, 3),
-			i64rotr: cost_instr!(instr_i64rotr, 3),
-			_phantom: PhantomData,
-		}
+		Self { base: cost_instr!(instr_i64const, 1), _phantom: PhantomData }
```

Comment:
can we inline the cost_instr macro since it's just used here now

Reply:
we could, but it really reads nicer like this, isn't it?